src/lib/server.rs

```diff
@@ -485,7 +486,8 @@ impl ServerClient {
         match self
             .client
             .get(format!("{}/words", self.api))
-            .query(request)
+            .header(ACCEPT, HeaderValue::from_static("application/json"))
+            .form(request)
             .send()
```

Comment:
Unlike others, this endpoint is using GET and accepts parameters.

So it would be a strange API design to use GET + form (so send application/x-www-form-urlencoded)

It is confirmed by the openapi spec

https://languagetool.org/http-api/#/default

https://languagetool.org/http-api/languagetool-swagger.json

![Screenshot_20250221_024557_Firefox.jpg](https://github.com/user-attachments/assets/7ccccf26-22ef-4ae9-b79f-6862990cbb44)

GET /words is the only one to use query 

So I think this shouldn't be changed. Or at least it should be triple-check. If the server supports it or not.

So I wouldn't change GET /words except the accept header.

Reply:
Hi @ccoVeille, thanks for your valuable comment! I confirm that this is also what I suppose: this should be the only `query` request, others use `formData`.

Reply:
No worries, thanks :-)